janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/inmemory/InMemoryKeyColumnValueStore.java

```diff
@@ -24,24 +25,43 @@
 import org.janusgraph.diskstorage.keycolumnvalue.*;
 import org.janusgraph.diskstorage.util.RecordIterator;
 import org.apache.commons.lang.StringUtils;
+import org.janusgraph.diskstorage.util.StaticArrayBuffer;
 
 import javax.annotation.Nullable;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
 import java.util.concurrent.ConcurrentNavigableMap;
 import java.util.concurrent.ConcurrentSkipListMap;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.ForkJoinTask;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.zip.Deflater;
+import java.util.zip.DeflaterOutputStream;
+import java.util.zip.Inflater;
+import java.util.zip.InflaterInputStream;
+
+import static java.util.zip.Deflater.BEST_SPEED;
+import static org.janusgraph.diskstorage.StaticBuffer.ARRAY_FACTORY;
 
 /**
  * An in-memory implementation of {@link KeyColumnValueStore}.
```

Comment:
i think you should update this comment. you changed the implementation.

Reply:
done